hack/jenkins/job-configs/kubernetes-e2e.yaml

```diff
@@ -21,14 +21,18 @@
             {provider-env}
             {job-env}
             {post-env}
-            {runner}
+            timeout -k {kill-timeout}m {timeout}m {runner}
```

Comment:
if this fails, does Jenkins immediately abort?

if not, should we instead be saving the exit code to use later?